integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java

```diff
@@ -169,7 +169,7 @@
 
   // Apache constants
   public static final String APACHE_IMAGE_NAME = "phx.ocir.io/weblogick8s/oracle/apache";
```

Comment:
Is it not available in OCR. When customer run the tests, they many not have access to OCIR.  It is an internal repository.

Reply:
The apache image is not in OCR. I built the image myself. I can ask Monica again whether we have official Apache image.